{"exporting_data_failed":{"attempt":5,"errors":["error while executing with response: [42000] [Simba][Hardy] (80) Syntax or semantic analysis error thrown in server while executing query. Error message from server: org.apache.hive.service.cli.HiveSQLException: Error running query: com.databricks.sql.managedcatalog.acl.UnauthorizedAccessException: PERMISSION_DENIED: User does not have READ FILES on External Location ***.\tat org.apache.spark.sql.hive.t (80) (SQLExecDirectW)"]}}
{"creating_remote_schema_failed":{"attempt":2,"errors":["error while executing with response: [42000] [Simba][Hardy] (80) Syntax or semantic analysis error thrown in server while executing query. Error message from server: org.apache.hive.service.cli.HiveSQLException: Error running query: java.lang.SecurityException: User does not have permission CREATE on CATALOG.\tat org.apache.spark.sql.hive.thriftserver.HiveThriftServerErrors$.runningQueryError(HiveThriftServerErrors.sc (80) (SQLExecDirectW)"]}}
